README.md

```diff
@@ -1,2 +1,68 @@
+
 # react-native-shopify
-React Native bridge to Shopify Buy SDK
+
+## Getting started
+
+`$ npm install react-native-shopify --save`
+
+### Including Mobile-BUY-SDK
+
+Include the Shopify Mobile Buy SDK in your project to make it available to the bridge. 
+Follow the instructions on their Github page to get started. For example, 
+the recommended and easiest way for iOS is to install it as a Pod. This project will look 
+for headers in the Pods directory. 
+
+### Mostly automatic installation
```

Comment:
This is an auto generated README from:

https://github.com/frostney/react-native-create-library

I just added the comment about including Mobile Buy SDK in the project.